@headlessui/tailwindcss incompatible to tailwindcss v4

**What package within Headless UI are you using?**

@headlessui/tailwindcss and @headlessui/vue

**What version of that package are you using?**

@headlessui/tailwindcss v0.2.1 and @headlessui/vue v1.7.23

**What browser are you using?**

Any

**Reproduction URL**

Just try to install tailwindcss v4 and @headlessui/tailwindcss:

```sh
% npm install -D tailwindcss@4

added 1 package in 175ms
% npm install @headlessui/tailwindcss
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: undefined@undefined
npm error Found: tailwindcss@4.0.3
npm error node_modules/tailwindcss
npm error   dev tailwindcss@"^4.0.3" from the root project
npm error
npm error Could not resolve dependency:
npm error peer tailwindcss@"^3.0" from @headlessui/tailwindcss@0.2.1
npm error node_modules/@headlessui/tailwindcss
npm error   @headlessui/tailwindcss@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /Users/topical/.npm/_logs/2025-02-06T11_18_51_142Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/topical/.npm/_logs/2025-02-06T11_18_51_142Z-debug-0.log
```

**Describe your issue**

@headlessui/tailwindcss is (officially) incompatible to tailwindcss v4

--- a/packages/@headlessui-tailwindcss/CHANGELOG.md
+++ b/packages/@headlessui-tailwindcss/CHANGELOG.md
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Nothing yet!
+- Support installing with Tailwind CSS v4 ([#3634](https://github.com/tailwindlabs/headlessui/pull/3634))
 
 ## [0.2.1] - 2024-05-29
 

--- a/packages/@headlessui-tailwindcss/package.json
+++ b/packages/@headlessui-tailwindcss/package.json
@@ -41,7 +41,7 @@
     "clean": "rimraf ./dist"
   },
   "peerDependencies": {
-    "tailwindcss": "^3.0"
+    "tailwindcss": "^3.0 || ^4.0"
   },
   "devDependencies": {
     "tailwindcss": "^3.2.7"

